pkg/agent/controller/networkpolicy/packetin.go

```diff
@@ -147,8 +156,32 @@ func (c *Controller) logPacket(pktIn *ofctrl.PacketIn) error {
 		return fmt.Errorf("received error while handling packetin for NetworkPolicy: %v", err)
 	}
 
-	// Store log file
-	AntreaPolicyLogger.Printf("%s %s %s %s SRC: %s DEST: %s %d %s", ob.tableName, ob.npRef, ob.disposition, ob.ofPriority, ob.srcIP, ob.destIP, ob.pktLength, ob.protocolStr)
+	// Create 1 sec buffers for deduplicating non-Allow packet log.
+	logMsg := fmt.Sprintf("%s %s %s %s SRC: %s DEST: %s %d %s", ob.tableName, ob.npRef, ob.disposition, ob.ofPriority, ob.srcIP, ob.destIP, ob.pktLength, ob.protocolStr)
+	if ob.disposition == openflow.DispositionToString[openflow.DispositionAllow] {
+		AntreaPolicyLogger.Printf(logMsg)
+	} else {
+		_, ok := logDeduplication[logMsg]
+		// Increase count and renew buffer if duplicated within 1 sec, create buffer otherwise.
+		if ok {
+			logDeduplication[logMsg].count++
+			logDeduplication[logMsg].bufferTimer.Reset(time.Second)
```

Comment:
I don't think we should do this as this can introduce too much delay before this connection shows up in the log file (if we keep receiving a matching packet every 500ms for example). I believe that packets should be buffered for _a maximum of 1 second_.